great_expectations/sqlalchemy_compatibility_wrappers.py

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
Love the rename. What do you think about moving into the `great_expectations/compatibility/` folder as well? That way we can keep all of the compatibility related code together.

Reply:
oo I like it a lot. Will do now